### DISCUSSION TITLE
Why can't I see the repos that others have starred?

### DISCUSSION THREAD

**Discussion by hmmhmmhm** [community/community#66748] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

<img width="904" alt="스크린샷 2023-09-11 오후 6 09 11" src="https://github.com/community/community/assets/38493722/8c80f329-1dc4-4e81-83c7-0b1d351ca496">

Hey, it's great that you're improving the Feed, but it's not working.

I am currently not seeing any repo listings that others have pressed STAR on. I used to be able to get a list of 1000+ every time just by looking at other people's recently pressed STAR lists

GitHub you are stealing my most valuable time again. I like to grab a coffee after work and review the repos that other developers around me have discovered.

I was included in this A/B test last year, and I contacted GitHub to get it removed, but they've dragged me back in again.

Is it really that hard to align with consumer needs?
Is it really that hard to just add new features while keeping the old ones working well....?

**Comment by hmmhmmhm** [comment]
<img width="946" alt="스크린샷 2023-09-11 오후 6 18 20" src="https://github.com/community/community/assets/38493722/3ae8c2ab-3169-4428-8f03-c80926cb0d57">


What the hell is....

**Comment by iainelder** [comment]
There are more of us discussing the same problem here. https://github.com/orgs/community/discussions/66210

You can use this link to the old data until GitHub fixes the main feed. https://github.com/dashboard-feed
